Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -79,7 +79,7 @@ internal VertexPositionTexture[] BuildVertices(float depth)
                     continue;
 
                 var region = _map.GetTileRegion(tile.Id);
-                var point = GetTileLocationFunction()(tile);
+                var point = GetTileLocationFunction()(tile).ToVector2() + new Vector2(OffsetX, OffsetY);
```

Comment:
I converted the `GetTileLocationFunction()(tile)` to `Vector2` to keep the offset precisions, intead of casting the offsets from float to int.